leakcanary-android-proguard/src/main/java/com/squareup/leakcanary/proguard/LeakcanaryProguardPlugin.kt

```diff
@@ -0,0 +1,42 @@
+package com.squareup.leakcanary.proguard
+
+import com.android.build.gradle.AppExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class LeakcanaryProguardPlugin : Plugin<Project> {
+  override fun apply(project: Project) {
+    val extension = project.extensions.create("leakCanary", LeakCanaryProguardExtension::class.java)
+
+    val androidPluginHandler = { _: Plugin<*> ->
+      val android = project.extensions.getByType(AppExtension::class.java)
+
+      project.afterEvaluate {
+        android.applicationVariants.all { variant ->
+
+          if (extension.obfuscatedVariantNames?.contains(variant.name) == true) {
```

Comment:
Is  == true necessary ?

Reply:
I think it is because `extension.obfuscatedVariantNames?.contains(variant.name)` returns a nullable `Boolean?` so I have to explicitly check for `true`. Is there a better way to do such check?

Reply:
The only thing that needs to get better is me at kotlin :)